src/lib/components/Loader.tsx

```diff
@@ -0,0 +1,11 @@
+import styled, { css } from 'lib/theme'
+
+export const loadingOpacityMixin = css<{ $loading: boolean }>`
+  filter: ${({ $loading }) => ($loading ? 'grayscale(1)' : 'none')};
+  opacity: ${({ $loading }) => ($loading ? '0.4' : '1')};
+  transition: opacity 0.2s ease-in-out;
+`
+
+export const LoadingOpacityContainer = styled.div<{ $loading: boolean }>`
```

Comment:
I don't think this merits being a component. It seems more like a theme/css addition to me.

wdyt about moving it to `lib/css/loading.ts`, and only exporting the css. It would then be used by creating a new styled component out of whichever element needed it:

```
const LoadingH2 = themed<{ loading: boolean }>(ThemedText.H2)`
  ${loadingOpacityCss};
`
```

Reply:
Yeah - this is probably better because it avoids us creating unneccesary container divs

Reply:
one issue im having though is styled in our theme wont let us import css like this into a styled component

Reply:
One argument for this though is that we end up writing styled components in every file that needs a loading state instead of referring to one component

Reply:
Pushed CSS version for now though